shapeshift/web
# feat: switch evm chain on contract call

## Description

Switches EVM chain on contract calls - note `chain-adapters` and `types` have been updated to the relevant version only, `chain-adapters` is still [behind](https://github.com/shapeshift/lib/releases) from the latest published version including osmosis lp changes.

## Notice




## Pull Request Type


## Issue (if applicable)

N/A

## Risk

None



## Testing



- See testing steps of https://github.com/shapeshift/web/pull/2348 - same baseline steps
- Ensure that chain switch (and chain addition as a preliminary step) if not already present in wallet) are working on any non-send Tx

### Engineering



☝🏽 

### Operations



☝🏽 

## Screenshots (if applicable)

<img width="392" alt="image" src="https://user-images.githubusercontent.com/17035424/210101469-9aff9149-8a51-4f3d-95b9-8de8198f6c67.png">
<img width="402" alt="image" src="https://user-images.githubusercontent.com/17035424/210101479-96be61de-8706-4308-b4dd-33cdac6446b2.png">
<img width="771" alt="image" src="https://user-images.githubusercontent.com/17035424/210101516-4e04d007-9187-42c6-890c-8e76a1d37d4a.png">
<img width="365" alt="image" src="https://user-images.githubusercontent.com/17035424/210101540-16491792-e11f-454a-bddb-f0c07fbbe654.png">
<img width="398" alt="image" src="https://user-images.githubusercontent.com/17035424/210101566-f2864370-a05d-46c9-ab58-c90a18b85c88.png">
